pydoctor/test/epydoc/test_restructuredtext.py

```diff
@@ -26,3 +29,80 @@ def test_rst_anon_link_email() -> None:
     assert html.startswith('<a ')
     assert ' href="mailto:postmaster@example.net"' in html
     assert html.endswith('>mailto:postmaster@example.net</a>')
+
+# THIS IS HELPER METHODS
+from pydoctor.epydoc.markup import flatten, restructuredtext
+def to_html(docstring) -> str:
+    """
+    Utility method to convert a docstring to html with pydoctor. 
+    """
+    err = []
+    p = restructuredtext.parse_docstring(docstring, err)
+    if [ e.__dict__ for e in err ]!=[]: 
+        raise ValueError("\n".join([ pprint.pformat(e.__dict__) for e in err ]))
+    html=flatten(p.to_stan(None))
+    return html
+
+def parse_and_print(s):
```

Comment:
```suggestion
def parse_and_print(s: str) -> None:
```

Reply:
If this function is here to stay, it would be best to remove it from `restructuredtext.doctest` and import it from this test module instead, to avoid code duplication.

Reply:
Or maybe the doctest should be converted into regular test cases. While the doctests are fairly readable, they lack the flexibility of being able to execute individual test cases and also the reporting when a test fails nowhere near as useful as pytest's usual reporting.